pages/deposits_and_withdrawals/cli_commands.md

```diff
@@ -0,0 +1,27 @@
+## CLI commands
+
+### Deposit to Subaccount
+
+```bash
+dydxprotocold tx sending deposit-to-subaccount <sender_key_or_address> <recipient_address> <recipient_subaccount_number> <quantums> [flags]
+```
+
+Example:
```

Comment:
also, can we add a line explaining that this is a "self-deposit"?

meaning that this deposits user's USDC from x/bank account to the x/subaccount account?

Reply:
This doesn't need to be a self deposit. You can deposit funds to a different address subaccount